suricata.yaml.in

```
@@ -1677,6 +1677,11 @@ detect:
     toserver-groups: 25
   sgh-mpm-context: auto
   inspection-recursion-limit: 3000
+  # try to tie an app-layer transaction for rules without app-layer keywords
+  # if there is only one live transaction for the flow
+  # allows to log app-layer metadata in alert
+  # but the transaction may not be the relevant one.
```

Tried to add a bit more here (from https://github.com/OISF/suricata/pull/12260/files):
```suggestion
  # Try to guess an app-layer transaction for rules without app-layer keywords,
  # ONLY IF there is just one live transaction for the flow.
  # This allows logging app-layer metadata in alert - the transaction may not
  # be the relevant one for the alert.
```

---

> Tried to add a bit more here (from https://github.com/OISF/suricata/pull/12260/files):

On suggestion...

> ONLY IF there is a single live transaction for the flow,

IMO, can be cleaned up in post-release.